cytotable/utils.py

```diff
@@ -453,3 +399,51 @@ def _expand_path(
         modifed_path = modifed_path.expanduser()
 
     return modifed_path.resolve()
+
+
+def _get_cytotable_version() -> str:
+    """
+    Seeks the current version of CytoTable using either pkg_resources
+    or dunamai to determine the current version being used.
+
+    Returns:
+        str
+            A string representing the version of CytoTable currently being used.
+    """
+
+    import dunamai
```

Comment:
Does this add a new dependency? It's probably ok and you've thought through this, but I typically get a bit itchy when this might happen!

Is there a world in which this import happens in the try statement? Or perhaps that would get too complicated too fast to handle if dependency doesn't exist?

Reply:
Great point, thanks! I'll update dunamai to be a dev-dependency only and catch the import error in addition to any runtime errors.

Reply:
Further clarity, dunamai appears to be the good way for now of adding developer / source-based version specification without too much trouble in development environments. For other scenarios (such as PyPI builds), the __version__ variable will be pre-populated with data accurate to the relevant release.